### PR TITLE
fix(lint): Remove TSLint leftovers

### LINKED ISSUE #253: Fix prepublish script
While moving from from TSLint from ESLint, the prepublish task broke - this needs to be fixed

### PR DESCRIPTION
**Pull request by Retro64**
close #253

### CODE REVIEW

**Comment by Retro64** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -69,7 +68,7 @@
     "lint": "eslint -c .eslintrc.js --ext .ts ./src/ ./test/",
     "lint:fix": "eslint --fix -c .eslintrc.js --ext .ts ./src/ ./test/",
     "build": "rm -rf lib && tsc",
-    "prepublishOnly": "npm run build && npm run test && npm run tslint"
+    "prepublishOnly": "npm run build && npm run test && npm run lint"
```
Comment (ISO50) [inline package.json:71]: Question, is the eslint based task also stopping the `publish` action?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "authmosphere",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (ISO50) [inline package.json:3]: We could think of adding a comment on the change log, I think we missed than in the previous PR.

**Comment by ISO50** [comment]
👍

File: package.json (hunk 1 above)
Comment (Retro64) [reply package.json:71]: yes - by intention - simply replacing tslint

**Comment by Retro64** [comment]
:+1:

File: package.json (hunk 2 above)
Comment (Retro64) [reply package.json:3]: done

**Comment by ISO50** [comment]
👍

**Comment by bzums** [comment]
👍

**Comment by bzums** [comment]
Build on two old node version fails, maybe we have to adjust the target version of the typescript compiler? But did not look into it yet.

**Comment by Retro64** [comment]
:+1:

**Comment by bzums** [comment]
👍

**Comment by Retro64** [comment]
:+1:

**Comment by Retro64** [comment]
:+1:

**Comment by bzums** [comment]
👍

**Pull request merged**